src/mail.sh

```diff
@@ -388,25 +662,45 @@ function parse_mail_options()
         options_values['SETUP']=1
         shift
         ;;
-      --email | --name)
-        validate_setup_opt "$1" || exit 95 # ENOTSUP
+      --email)
+        if [[ -z "${options_values['sendemail.smtpuser']}" ]]; then
+          options_values['sendemail.smtpuser']="$2"
+          smtpuser_autoset=1
+        fi
+        ;&
```

Comment:
iirc case has different types of stops, is this one of them? Or is it a bug?

Reply:
Yes, this is the "Also execute the next case". This way the code shared by `name` and `email` still runs.

Reply:
Since this is not common on kw code base, could you just add one comment next to it?